server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java

```diff
@@ -566,4 +574,12 @@ Set<String> getSupportedCipherSuites() {
     }
     return Sets.newHashSet(context.getServerSocketFactory().getSupportedCipherSuites());
   }
+
+  private static Map<String, String> overwriteJettyDefaultConfig(
+      Map<String, String> properties, OverwriteDefaultConfig config) {
+    if (config.getOverwriteDefaultConfig().isEmpty()) return properties;
+    Map<String, String> newProperties = new HashMap<>(properties);
+    config.getOverwriteDefaultConfig().forEach((k, v) -> newProperties.putIfAbsent(k, v));
```

Comment:
Why use this `putIfAbsent`, I would assume that we should use put to overwrite?

Reply:
If user set IcebergRESTservice http port explictly, we should not overwrite it

Reply:
I see.